lr-cli/cortx_setup/commands/storage/config.py

```diff
@@ -116,17 +116,17 @@ class StorageEnclosureConfig(Command):
             'help': 'Port of the controller to connect to'
         },
         'cvg': {
-            'type': int,
-            'default': -1,
+            'type': str,
+            'default': None,
             'optional': True,
-            'help': 'Cylinder Volume Group number.'
+            'help': 'Disk group name'
         },
-        'data_devices': {
+        'data-devices': {
             'type': str,
              'optional': True,
             'help': 'List of data devices (Comma separated) e.g /dev/mapper/mpatha,/dev/mapper/mpathb'
         },
-        'metadata_devices': {
+        'metadata-devices': {
             'type': str,
             'optional': True,
             'help': 'List of metadata devices (Comma separated) e.g /dev/mapper/mpathf,/dev/mapper/mpathg'
```

Comment:
Lets change help message:
`List of metadata devices mapped to CVG disk-group E.g. /dev/mapper/mpathx,/dev/mapper/mpathy`

Reply:
Done